mne/viz/_3d.py

```diff
@@ -1344,3 +1345,135 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_3d(dipole, trans, subject, subjects_dir=None, scale_factor=1e9,
+                   block=False, show=True):
+    """Plot dipoles in 3-D. Browse through the dipoles using mouse scroll or
+    up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
+    trans : dict
+        The mri to head trans.
+    subject : str
+        The subject name corresponding to FreeSurfer environment variable
+        SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. The default is None.
```

Comment:
`subjects_dir` is usually toward the end, especially since many people don't need to explicitly set this (it's part of the env vars or config vars)